cache/blobs.go

```diff
@@ -106,13 +108,37 @@ func computeBlobChain(ctx context.Context, sr *immutableRef, createIfNeeded bool
 			if release != nil {
 				defer release()
 			}
-			desc, err := sr.cm.Differ.Compare(ctx, lower, upper,
-				diff.WithMediaType(mediaType),
-				diff.WithReference(sr.ID()),
-				diff.WithCompressor(compressorFunc),
-			)
-			if err != nil {
-				return nil, err
+			var desc ocispecs.Descriptor
+			if len(lower) > 0 && !isTypeWindows(sr) {
```

Comment:
`BUILDKIT_DEBUG_FORCE_OVERLAY_DIFF=0` should also be a condition to skip this. This can be follow-up to the time when tests would actually set it.